lib/utils.js

```diff
@@ -1,13 +1,34 @@
 var assert = require('assert');
 var InvalidConfigError = require('./errors/InvalidConfigError');
 var legalChars = new RegExp('^[a-zA-Z0-9._-]*$');
+var allowedTopicLength = 249;
 
 function validateConfig (property, value) {
   if (!legalChars.test(value)) {
     throw new InvalidConfigError([property, value, "is illegal, contains a character other than ASCII alphanumerics, '.', '_' and '-'"].join(' '));
   }
 }
 
+function validateCreateTopics (topics) {
```

Comment:
Maybe rename this to `validateTopicNames` since this isn't specific to just creating topics but we can reuse this to validate topics in the other classes.